web_view_leaflet_map/__manifest__.py

```diff
@@ -4,8 +4,7 @@
 
 {
     "name": "Leaflet Map View (OpenStreetMap)",
-    "summary": "Integrate leaflet.js librairy with odoo"
-    " and add new 'leaflet_map' view, to display markers.",
+    "summary": "Add new 'leaflet_map' view, to display markers.",
     "version": "16.0.1.0.0",
```

Comment:
Version to raise?

Reply:
> Version to raise?


Sorry. I don't understand.

Reply:
I get it ! Done. 

Can be merged now, @yvaucher . Thanks for your review !